atom/common/api/atom_bindings.cc

```diff
@@ -125,6 +126,35 @@ void AtomBindings::Hang() {
     base::PlatformThread::Sleep(base::TimeDelta::FromSeconds(1));
 }
 
+// static
+v8::Local<v8::Value> AtomBindings::GetHeapStatistics(v8::Isolate* isolate) {
+  v8::HeapStatistics v8_heap_stats;
```

Comment:
"v8HeapStats" to match electron naming.

Reply:
There are different rules for c++ variable names